README.md

```diff
@@ -1 +1,20 @@
-"# DeathDB" 
+# DeathDB
+
+## About
+
+DeathDB is a plugin that allows Staff Members to log data about Player Deaths.
+
+When a player dies, DeathDB will log the following information to a Table with the name of their UUID in
+an SQLite Database:
+
+- The location (X, Y, Z) of the death.
+- The name of the world the player died in.
+- A serialized copy of the players inventory.
+
+This information can then be retrieved at a later date by Staff Members. They also have the option of
+restoring the players inventory, if they deem necessary.
+
+## Compilation
+
+Compilation Steps are currently unavailable until a production release of the plugin can be made. Please
+check back at a later date, when the plugin has been further developed.
```

Comment:
If it is ready, this should be updated :)

Reply:
Will do ASAP.

Reply:
Compilation steps added, will push shortly.